docs/reference/msbuild-targets.md

```diff
@@ -59,8 +59,10 @@ Note that the `Owners` and `Summary` properties from `.nuspec` are not supported
 | IconUrl | PackageIconUrl | empty | |
 | Tags | PackageTags | empty | Tags are semi-colon delimited. |
 | ReleaseNotes | PackageReleaseNotes | empty | |
-| RepositoryUrl | RepositoryUrl | empty | |
-| RepositoryType | RepositoryType | empty | |
+| Repository/Url | RepositoryUrl | empty | Repository URL used to clone or retrieve source code. Example: *https://github.com/NuGet/NuGet.Client.git* |
+| Repository/Type | RepositoryType | empty | Repository type. Examples: *git*, *tfs*. |
+| Repository/Branch | RepositoryBranch | empty | Optional repository branch information. Example: *master* (NuGet 4.7.0+) |
```

Comment:
can we add that this will only be picked up if url is provided?

Reply:
Added